test/common/chromium-cdp-driver.js

```diff
@@ -279,60 +274,59 @@ function setupSSLCertHandling(client) {
  *    fail <JSON formatted error>
  * EOF shuts down the client.
  */
-process.stdin.setEncoding('utf8');
+async function main() {
+    process.stdin.setEncoding('utf8');
+
+    if (process.env.TEST_CDP_DEBUG)
+        enable_debug = true;
+
+    const options = { };
+    if (process.argv.length >= 3) {
+        options.port = parseInt(process.argv[2]);
+        if (!options.port) {
+            process.stderr.write("Usage: chromium-cdp-driver.js [port]\n");
+            process.exit(1);
+        }
+    }
 
-if (process.env.TEST_CDP_DEBUG)
-    enable_debug = true;
+    const target = await CDP.New(options);
+    target.port = options.port;
+    const client = await CDP({ target });
+    setupLogging(client);
+    setupFrameTracking(client);
+    setupSSLCertHandling(client);
+    setupLocalFunctions(client);
+
+    let input_buf = '';
+    for await (const chunk of process.stdin) {
```

Comment:
@allisonkarlitskaya brought up the idea of a line iterator, which would get rid of that buffer processing. Need to check if that's possible.

Reply:
https://nodejs.org/api/readline.html

Reply:
Oh yeah, much nicer!